src/fluree/server/handlers/shared.clj

```diff
@@ -11,6 +11,7 @@
       res)))
 
 (defmacro defhandler
+  {:clj-kondo/lint-as 'clojure.core/defn}
```

Comment:
there might not be a way around it, but i'm not a big fan of annotations for external tooling in code. i'd prefer that this kind of stuff go in a separate config file if possible.

Reply:
Yeah I think there's a way to do that. I'll look into it.